File: robot/MotorTest.py
import RPi.GPIO as GPIO
from time import sleep

print('start')

rf1 = 13
rf2 = 15
rfe = 21

GPIO.setmode(GPIO.BOARD)

GPIO.setup(rfe, GPIO.OUT)
GPIO.setup(rf1, GPIO.OUT)
GPIO.setup(rf2, GPIO.OUT)

GPIO.output(rfe, GPIO.HIGH)
GPIO.output(rf1, GPIO.HIGH)
GPIO.output(rf2, GPIO.LOW)

sleep(3)

print('we made it')
GPIO.output(rfe, GPIO.LOW)

GPIO.cleanup()

File: robot/MotorTest.js
var Gpio = require('onoff').Gpio
,   rf1  = new Gpio(23, 'out')
,   rf2  = new Gpio(24, 'out')
,   rfe  = new Gpio(25, 'out')
;

console.log('start');

function check(err) {
    if (err) console.log(err);
    else console.log('woo');
}

rfe.write(1, check);
rf1.write(0, check);
rf2.write(1, check);

setTimeout(function () {
    rfe.unexport();
    rf1.unexport();
    rf2.unexport();

    console.log('we made it');
}, 3000);

